autoload/goyo.vim

```diff
@@ -33,8 +33,12 @@ function! s:get_color(group, attr)
 endfunction
 
 function! s:set_color(group, attr, color)
-  let gui = a:color =~ '^#'
-  execute printf('hi %s %s%s=%s', a:group, gui ? 'gui' : 'cterm', a:attr, a:color)
+  if empty(a:attr)
```

Comment:
`s:set_color(grp, 'bg', 'NONE')` is a `call` that changes the bg of the highlight to none, while `if empty(a:attr)` targets the decorations of the highlight. You wouldn't want `if a:color == 'NONE'` because it will catch the expression that will turn the bg to `none`, and will fail to turn the bg to `none`.